next.config.js

```diff
@@ -1,6 +1,13 @@
-const withNextra = require('nextra')({
-  theme: 'nextra-theme-docs',
-  themeConfig: './theme.config.js',
+const withNextra = require("nextra")({
+  theme: "nextra-theme-docs",
+  themeConfig: "./theme.config.js",
+  unstable_flexsearch: true,
   unstable_staticImage: true,
-})
-module.exports = withNextra()
+});
+
+module.exports = withNextra({
+  // reactStrictMode: true,
```

Comment:
it's better to enable strict mode
```suggestion
  reactStrictMode: true,
```